ComponentKit/TransactionalDataSources/Common/CKTransactionalComponentDataSource.mm

```diff
@@ -45,6 +58,7 @@ - (instancetype)initWithConfiguration:(CKTransactionalComponentDataSourceConfigu
     _announcer = [[CKTransactionalComponentDataSourceListenerAnnouncer alloc] init];
     _workQueue = dispatch_queue_create("org.componentkit.CKTransactionalComponentDataSource", DISPATCH_QUEUE_SERIAL);
     _pendingAsynchronousModifications = [NSMutableArray array];
+    _workThreadOverride = configuration.workThreadOverride;
```

Comment:
Decided against allowing the work thread override to be changed after the transactional data source is initialized.